code/modules/admin/topic.dm

```diff
@@ -982,7 +982,8 @@
 
 	else if(href_list["removenote"])
 		var/note_id = href_list["removenote"]
-		remove_note(note_id)
+		if(alert("Do you really want to delete this note?","Note deletion confirmation", "Yes", "No") == "Yes")
```

Comment:
```suggestion
		if(alert("Do you really want to delete this note?", "Note deletion confirmation", "Yes", "No") == "Yes")
```

Reply:
ah thanks, there is always something to find eh :P